brave/brave-core
# Cleanup Brave Ads return by const &

Resolves https://github.com/brave/brave-browser/issues/24611

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Confirm users are rewarded for ad impressions (only need to test a single creative ad type)
- Confirm confirmation tokens are redeemed after a retry
- Confirm toggles are working as expected on brave://rewards 30-day ads history
- Confirm subdivision targeting is working as expected
- Confirm that unblinded payment tokens are redeemed (cashed-out)
